quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/main.bicep

```diff
@@ -0,0 +1,126 @@
+// New Resources Name
```

Comment:
remove comment since the description should be explanatory

Reply:
Removed comment